lib/src/main/java/rife/authentication/remembermanagers/DatabaseRemember.java

```diff
@@ -46,8 +50,20 @@ public abstract boolean remove()
     throws RememberManagerException;
 
     protected boolean _install(CreateTable createRemember, String createRememberMomentIndex) {
-        executeUpdate(createRemember);
-        executeUpdate(createRememberMomentIndex);
+        try {
```

Comment:
The framework doesn't use transactions for database structure creation, since some databases don't support this. If you look at the `_install` and `_remove` methods of other data managers in the authentication framework, you'll see how I do it there.